content/docs/hooks-reference.md

```diff
@@ -56,18 +56,41 @@ If the new state is computed using the previous state, you can pass a function t
 ```js
 function Counter({initialCount}) {
   const [count, setCount] = useState(initialCount);
+
+  function incrementBy(x) {
+    setCount((prevCount) => prevCount + x);
+  }
+
+  function multiplyBy(x) {
+    setCount((prevCount) => prevCount * x);
+  }
+
   return (
     <>
       Count: {count}
       <button onClick={() => setCount(initialCount)}>Reset</button>
-      <button onClick={() => setCount(prevCount => prevCount - 1)}>-</button>
```

Comment:
The `setCount` here takes an arrow function as param (`functional form`) since the state update depends on the previous state value. Whereas the reset action does not depend on the previous value and it is going to reset the state value so we do not use the functional form here.


`If the new state is computed using the previous state, you can pass a function to setState. The function will receive the previous value, and return an updated value. `
https://reactjs.org/docs/hooks-reference.html#functional-updates